ztp/du-profile-kube-compare/required/node-tuning-operator/PerformanceProfile.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  # if you change this name make sure the 'include' line in TunedPerformancePatch.yaml
+  # matches this name: include=openshift-node-performance-${PerformanceProfile.metadata.name}
+  # Also in file 'validatorCRs/informDuValidator.yaml': 
+  # name: 50-performance-${PerformanceProfile.metadata.name}
+  name: openshift-node-performance-profile
+  annotations:
+    ran.openshift.io/reference-configuration: "ran-du.redhat.com"
+spec:
+  additionalKernelArgs:
+  - "rcupdate.rcu_normal_after_boot=0"
+  - "efi=runtime"
+  - "vfio_pci.enable_sriov=1"
+  - "vfio_pci.disable_idle_d3=1"
+  - "module_blacklist=irdma"  
+  cpu:
+    isolated: {{ .spec.cpu.isolated }}
+    reserved: {{ .spec.cpu.reserved }}
+  hugepages:
+    defaultHugepagesSize: {{ .spec.hugepages.defaultHugepagesSize }}
+    pages:
+      {{- range .spec.hugepages.pages }}
+      - size: {{ .size  }}
+        count: {{ .count }}
+        {{- if .node }}
+        node: {{ .node }}
+        {{- end }}
+      {{- end }}
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/master: ""
+  nodeSelector:
+    node-role.kubernetes.io/master: ''
+  numa:
+    topologyPolicy: "restricted"
+  # To use the standard (non-realtime) kernel, set enabled to false
+  realTimeKernel:
+    enabled: true
```

Comment:
Real time kernel is optional. Can be set to false

Reply:
I don't see the update. The realtime kernel is still set to "enabled: true" as the only option.